docs/architecture/application/graphql.md

```diff
@@ -47,12 +55,68 @@ class NewQuery(Query):
     )
     new: str = strawberry.field(resolve_new, description="new resolver")
 
+# without customers.
+@strawberry.federation.type(description="Orchestrator queries")
+class NewQueryWithoutCustomers(OrchestratorQuery):
+    other_processes: Connection[ProcessType] = authenticated_field(
+        resolver=resolve_processes,
+        description="resolve_processes used for another field",
+    )
+    new: str = strawberry.field(resolve_new, description="new resolver")
+
 
 app = OrchestratorCore(base_settings=AppSettings())
 # register SUBSCRIPTION_MODEL_REGISTRY
 app.register_graphql(query=NewQuery)
 ```
 
+## Adding federated types to the graphql
+
+Within a federation, it is possible to add orchestrator data to graphql types from other sources by extending the `DEFAULT_GRAPHL_MODELS` dictionary with your own federated classes and adding them as parameter to `app.register_graphql(graphql_models={}). Here is an example for when instead of overriding the customers resolver, you instead use a different graphql source (know that not storing any customer data in the orchestator will make filtering and sorting unavailable and very tricky to implement):
```

Comment:
```suggestion
Within a federation, it is possible to add orchestrator data to graphql types from other sources by extending the `DEFAULT_GRAPHL_MODELS` dictionary with your own federated classes and adding them as parameter to `app.register_graphql(graphql_models={})`. Here is an example for when instead of overriding the customers resolver, you instead use a different graphql source (know that not storing any customer data in the orchestator will make filtering and sorting unavailable and very tricky to implement):
```